apps/frontend/src/hooks/useGetNativeTokenPrice.ts

```diff
@@ -2,33 +2,34 @@ import { useMemo } from 'react';
 
 import axios from 'axios';
 
+import { ChainId } from '@sovryn/ethers-provider';
+
+import { RSK_CHAIN_ID } from '../config/chains';
+
 import { BITCOIN, ETH } from '../constants/currencies';
 import { COMMON_SYMBOLS } from '../utils/asset';
 import { isRskChain } from '../utils/chain';
 import { getIndexerUrl } from '../utils/helpers';
 import { toWei } from '../utils/math';
 import { useCacheCall } from './useCacheCall';
-import { useCurrentChain } from './useChainStore';
 import { useDollarValue } from './useDollarValue';
 
-export const useGetNativeTokenPrice = () => {
-  const currentChainId = useCurrentChain();
-
+export const useGetNativeTokenPrice = (chainId: ChainId) => {
```

Comment:
```suggestion
export const useGetNativeTokenPrice = (chainId?: ChainId) => {
```
This should be only optional and you can use either `chainId` if provided or `currentChainId` if not. Otherwise, you're forced to change all the occurrences and you usually don't want to override `currentChainId`.